target/linux/mediatek/image/mt7622.mk

```diff
@@ -176,6 +176,40 @@ define Device/linksys_e8450-ubi
 endef
 TARGET_DEVICES += linksys_e8450-ubi
 
+define Device/iptime_ax8004m
```

Comment:
I think it makes more sense in this case to add only a single variant of this device, similar to how we are doing it for the Xiaomi AX6S device (https://github.com/openwrt/openwrt/pull/4810).
It would of course be nice (imho) to have a fully UBInized version of the firmware, like we do for the E8450/RT3200, ie. replacing the bootloader and storing **everything** in UBI. That would then also justify having a separate non-UBI variant of the image to be build.
However, here you are using UBI only half-way while still using a BMT-enabled bootchain. This is also a valid approach and certainly better than just using JFFS2 on NAND (which we should **never** do in case it can be avoided by any means).